js/data/bucket.js

```diff
@@ -58,8 +58,6 @@ class Bucket {
                 options.featureIndex.insert(feature, this.index);
```

Comment:
@jfirebaugh Can you please clarify how this works correctly for symbols? Should featureIndex only have geometries that are actually rendered on the map? Would this (or not) add all point geometries as long as they match the filter?